CMakeLists.txt

```diff
@@ -57,16 +57,38 @@ set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 set(CMAKE_VISIBILITY_INLINES_HIDDEN TRUE)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 
+# Conan dependencies
+include(conan)
+set(DYNAMIC_LIB_TRUE_FALSE "False")
```

Comment:
Can't you just `set(DYNAMIC_LIB_TRUE_FALSE ${DYNAMIC_LIB})`?

Reply:
No, unfortunately it needs to be "True" or "False" specifically, not "ON" or any of the other options supported by cmake.